packages/atomic/src/components/atomic-search-box/atomic-search-box.tsx

```diff
@@ -200,50 +201,51 @@ export class AtomicSearchBox {
           onMouseDown={(e) => e.preventDefault()}
           part="suggestion"
           id={id}
-          class="suggestion h-9 px-2 cursor-pointer text-left text-sm bg-transparent border-none shadow-none hover:bg-primary hover:text-on-primary flex flex-row items-center"
+          class="suggestion h-9 px-2 cursor-pointer text-left text-sm bg-transparent border-none shadow-none flex flex-row items-center"
           innerHTML={suggestion.highlightedValue}
           value={index}
         />
       );
     });
   }
 
-  private renderInputWrapper() {
-    let roundedClasses = this.leadingSubmitButton
-      ? 'border-l-0 rounded-r-lg'
-      : 'border-r-0 rounded-l-lg';
-
-    if (this.searchBoxState.suggestions.length) {
-      roundedClasses += ' rounded-bl-none rounded-br-none';
-    }
-
-    return (
-      <div
-        part="input-wrapper"
-        class={`input-wrapper flex flex-grow items-center border border-divider ${roundedClasses}`}
-        ref={(el) => (this.containerRef = el as HTMLElement)}
-      >
-        {this.input}
-        {this.clearButton}
-      </div>
-    );
-  }
-
   public render() {
     return (
-      <div class="relative">
-        <div class="search-box box-border w-full flex">
+      <div class="search-box relative w-full lg:w-80">
+        <div
+          class={
+            'search-box-wrapper flex items-center focus:outline-none box-border w-full ' +
+            (this.searchBoxState.suggestions.length > 0 &&
+            this.shouldShowSuggestions
+              ? 'has-values'
+              : '')
+          }
+        >
           {this.leadingSubmitButton && this.submitButton}
-          {this.renderInputWrapper()}
+          <div class="combobox flex flex-grow h-full">
+            <div
+              class={
+                'input-wrapper flex flex-grow items-center h-full apply-border-on-background border-r-0 ' +
+                (this.leadingSubmitButton ? 'rounded-r-lg' : 'rounded-l-lg')
+              }
+              ref={(el) => (this.containerRef = el as HTMLElement)}
+            >
+              {this.input}
+              {this.clearButton}
+            </div>
+            <ul
+              part="suggestions"
+              class={
+                'suggestions absolute border-t-0 w-full bg-background apply-border-on-background empty:border-none rounded-b ' +
+                (this.shouldShowSuggestions ? 'block' : 'hidden')
```

Comment:
Its generally better for performance/accessibility to not render an element rather than render and hide it using CSS.
In this case, `this.shouldShowSuggestions` could be used to decide whether to include the `ul` in the DOM. e.g.

```
{this.shouldShowSuggestions && <ul ...>}
```